src/xdp-utils.c

```diff
@@ -884,6 +884,36 @@ verify_proc_self_fd (XdpAppInfo *app_info,
   return xdp_app_info_remap_path (app_info, path_buffer);
 }
 
+static char *documents_mountpoint = NULL;
+
+void
+xdp_set_documents_mountpoint (const char *path)
+{
+  g_clear_pointer (&documents_mountpoint, g_free);
```

Comment:
Why do we need to allow this variable to be set multiple times?

Reply:
It isn't necessary for xdg-desktop-portal or xdg-document-portal.  It turned out to be useful in writing the test, so I've left it as is.